builders/xcm/xcm-sdk/xcm-sdk.md

```diff
@@ -0,0 +1,849 @@
+| `destinationBalance` |                                                                                                                                        the balance of the asset being transferred on the destination chain                                                                                                                                         |
+|   `destinationFee`   |                                                                                                                                          the fee for the asset to be transferred on the destination chain                                                                                                                                          |
+| `existentialDeposit` | the [existential deposit](https://support.polkadot.network/support/solutions/articles/65000168651-what-is-the-existential-deposit-#:~:text=On%20the%20Polkadot%20network%2C%20an,the%20Existential%20Deposit%20(ED).){target=_blank}, or the minimum amount an address must <br> hold to be considered active if one exists, otherwise `0n` is returned |
+|        `min`         |                                                                                                                                                          the minimum transferrable amount                                                                                                                                                          |
+|       `native`       |                                                                                                                                                  the native [asset](#assets) of the source chain                                                                                                                                                   |
+|       `origin`       |                                                                                                                                the chain information for where the asset being transferred natively originates from                                                                                                                                |
+|       `getFee`       |                                                                                                                               a function that [estimates the fees](#get-fee-withdraw) for depositing a given amount                                                                                                                                |
+|        `send`        |                                                                                                                                            a function that [sends](#send-withdraw) the withdraw request                                                                                                                                            |
```

Comment:
@ekenigs Can you please review these definitions too? Same situation as the other section 😂